tests/python/unittest/test_tf_model.py

```diff
@@ -14,42 +14,38 @@ def _generate_frozen_graph():
     tf.square(mm, name="output1")
     mm_flat = tf.reshape(mm, shape=[-1])
     tf.argmax(mm_flat, name="output2")
-
     with tf.Session() as sess:
-        output_graph_def = tf.compat.v1.graph_util.convert_variables_to_constants(
+        output_graph_def = tf.graph_util.convert_variables_to_constants(
```

Comment:
nvidia only provides TF 1.10.0 for Jetson TX2
https://developer.download.nvidia.com/compute/redist/jp40/tensorflow-gpu/
```
Index of /compute/redist/jp40/tensorflow-gpu
..
tensorflow_gpu-1.10.0+nv18.9-cp27-cp27mu-linux_aarch64.whl 155MB 2018-09-14 14:23
tensorflow_gpu-1.10.0+nv18.9-cp36-cp36m-linux_aarch64.whl 155MB 2018-09-14 17:10
```
tf.compat.v1 was added after TF 1.10.0.
Because of that I switched to original deprecated API here.